app/components/Stepper.tsx

```diff
@@ -0,0 +1,59 @@
+import { Link, useLocation } from '@remix-run/react'
+import React from 'react'
+
+const steps = [
+  { label: 'Add your resume' },
+  { label: 'Add job offer' },
+  { label: 'Enhance resume' },
+]
+
```

Comment:
**suggestion**: Consider using a more descriptive variable name for `currentStep` to improve readability, such as `currentStepName`.